File: Deploy/deploy/test/project/dave_main.py
# -*- coding: utf-8 -*-
#/*
# * Copyright (c) 2022 Renwei
# *
# * This is a free software; you can redistribute it and/or modify
# * it under the terms of the MIT license. See LICENSE for details.
# */
import sys
import public


# =====================================================================


def main():
   product_name = sys.argv[1]

   public.dave_python_init(product_name)
   public.dave_python_running()
   public.dave_python_exit()
   return


if __name__ == "__main__":
   main()

File: Project/Go/project/dave_main.go
package main
/*
 * Copyright (c) 2022 Renwei
 *
 * This is a free software; you can redistribute it and/or modify
 * it under the terms of the MIT license. See LICENSE for details.
 */

import (
	"dave/public/base"
	"dave/product"
	"strings"
)

// =====================================================================

func main() {
	_, workmode := product.Product_cfg()

	base.Dave_go_init("", workmode, "", product.Product_init, product.Product_exit)
	if find := strings.Contains(workmode, "Outer"); find {
		base.Dave_go_running()
	}
	base.Dave_go_exit()
}